charts/lws/values.yaml

```diff
@@ -4,57 +4,48 @@
 
 nameOverride: ""
 fullnameOverride: ""
-
 enablePrometheus: false
 enableCertManager: false
-
 replicaCount: 1
 imagePullSecrets: []
-
 # Customize controlerManager
 image:
   manager:
     repository: us-central1-docker.pkg.dev/k8s-staging-images/lws/lws
     # tag, if defined will use the given image tag, else Chart.AppVersion will be used
     tag: main
     pullPolicy: IfNotPresent
-
+  repository: us-central1-docker.pkg.dev/k8s-staging-images/lws/lws
```

Comment:
From release task, why we have another repository here, I didn't find anywhere to use this in this configuration, but we do use the `image.manager.repository`.

Reply:
see https://github.com/kubernetes-sigs/lws/pull/481 for details.